appveyor.yml

```diff
@@ -2,7 +2,7 @@
 
 image: Visual Studio 2017
 
-# cache version - netcoreapp.2.0.0-preview3-25426-01
+# cache version - netcoreapp.2.0.0
 cache:
   - '%LocalAppData%\Microsoft\dotnet -> appveyor.yml'
```

Comment:
There are some changes pending for `PowerShell.Common.props` (see Ilya's MSBuild refactoring PRs). I think for the time being, `PowerShell.Common.props` has more churns than `appveyor.yml`.